test/apps/react-oie/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.0.0",
   "scripts": {
     "start": "vite",
-    "dev": "vite dev",
+    "dev": "NODE_ENV=development vite dev",
```

Comment:
vite dev serves native ESM to browser, this env var should not be needed.

Reply:
It was including runtime regenerator unless I set target to "esnext" (default is "modules")